coderd/coderd.go

```diff
@@ -518,3 +522,26 @@ func compressHandler(h http.Handler) http.Handler {
 
 	return cmp.Handler(h)
 }
+
+func entitlements(rw http.ResponseWriter, _ *http.Request) {
```

Comment:
opinion: I think calling this `nopEntitlements` would make it a little more obvious what this does on the AGPL side.